docs/tutorials/replicated-stateful-application/mysql-statefulset.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1beta1
```

Comment:
In my Vitess chart, I found it useful to have multiple StatefulSets share a single headless service. Is that a reasonable pattern? If so, the convention of putting the headless service together with the StatefulSet wouldn't make sense. For the tutorial, I also like the idea of keeping the service separate to reduce noise, and so I can separately explain the parts. If we still want to stick to the convention though, I'll do that. Let me know what you think.